Tests/LootLockerTestUtils/LootLockerTestConfigurationGame.cs

```diff
@@ -0,0 +1,311 @@
+﻿using LLlibs.ZeroDepJson;
+using LootLocker;
+using LootLocker.Requests;
+using System;
+using System.Collections.Generic;
+using Debug = UnityEngine.Debug;
```

Comment:
Are we actually using System? In my experience this happens when I accidentally include the wrong library for Debug.Log (if I recall correctly) which it seems like here with the explicit `using Debug = UnityEngine.Debug`

Reply:
Fixed